virttest/utils_conn.py

```diff
@@ -1166,7 +1170,7 @@ def server_setup(self, on_local=False):
         tmp_dir = self.tmp_dir
         scp_new_cacert = self.scp_new_cacert
         # sometimes, need to reuse previous CA cert
-        if self.ca_cakey_path and scp_new_cacert == 'no':
+        if self.ca_cakey_path:
```

Comment:
In my understanding, scp_new_cacert='no' means to reuse the previous CA cert, and ca_cakey_path provides the CA cert path.

Only ca_cakey_path can't meet the condition to reuse the previous CA cert. When I go through the code of build_CA(), I found when ca_cakey_path provides existing CA key, and new cert can be created with this existing CA key.